types/pb/sf/near/transform/v1/transform.pb.go

```diff
@@ -1,12 +1,13 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.25.0
```

Comment:
You seem to have used an oder version, upgrade to at least `v1.27.1` (and fix the `generate.sh` where you maybe took that).